packages/catalog-model/api-report.md

```diff
@@ -53,6 +53,16 @@ export { ApiEntityV1alpha1 };
 // @public
 export const apiEntityV1alpha1Validator: KindValidator;
 
+// @public (undocumented)
+export function areGroupEntities(
+  entities: Entity[],
+): entities is GroupEntityV1alpha1[];
```

Comment:
I'd prefer not to have these additions to the API actually. We are still figuring out what the best way forward is for proper usage of apiVersions. If we settle on a v1 of Group we'd be tempted to change the return type of these to that "new most wanted" version of a group potentially leading to breakages. Plus arguably the contract of the functions (acting on lists) is a bit oddly specific from an outsider's point of view.

Maybe just inline these privately where they are used?

Reply:
Oh haha, I wrote this before realizing we already have the one-entity version of the same already :) Well, I think the comment still stands, let's not dig this particular hole any deeper.

Reply:
cool, I'll do this locally to the plugin 👍

Reply:
done :)